src/Actions/UpdateTeamMemberRole.php

```diff
@@ -16,17 +17,17 @@ class UpdateTeamMemberRole
      * @param  mixed  $user
      * @param  mixed  $team
      * @param  int  $teamMemberId
-     * @param  string  $role
+     * @param  array<string>|string|null  $role
      * @return void
      */
-    public function update($user, $team, $teamMemberId, string $role)
+    public function update($user, $team, $teamMemberId, $role)
```

Comment:
Could rename these variables (and the one in the docblocks to `$roles`.

Reply:
Of course. I am trying to keep the changes to minimum to help the review process, and if you like it, I can add as many changes you like

Reply:
yeah feel free to rename these variables as well. Think that'd make more sense.

In the end though, it's @taylorotwell who decides about this.

Reply:
Did the rename on changed code, however it will need some more for consistency. (After I get 'official' acceptance ) :pray: